packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -122,40 +169,76 @@
     &__jurisdiction-checkbox {
         display: flex;
         justify-content: center;
+        margin-bottom: 2em;
         @include mobile {
             padding: 0.8rem;
         }
     }
 
     &__over-header {
         background-color: #dfeaff;
-        top: -1rem;
+        top: -2rem;
         left: 1.4rem;
         z-index: 10;
         width: 90%;
         height: 4rem;
         position: absolute;
-        border-radius: 4px;
+        border-radius: 8px;
         margin-bottom: 1rem;
-
+        border: 1px solid var(--text-info-blue);
         & > p {
             font-size: 16px;
             text-align: center;
         }
     }
+}
 
-    &__bullet-wrapper {
-        display: flex;
-        margin-bottom: 1rem;
+.cfd-card-section {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
 
-        &--checkmark {
-            margin-right: 1rem;
-        }
+.cfd-card-section-divider {
+    width: 100%;
+    border-bottom: 2px solid var(--general-section-1);
+}
+
+.cfd-card-section-title-indicator {
+    display: flex;
+    padding: 6px 12px;
+    color: var(--text-prominent);
+    border-radius: 4px;
+    font-weight: bold;
+    &__red-darker {
+        background: #661b20;
+    }
+    &__red-dark {
+        background: #b33037;
+    }
+    &__red-light {
+        background: #ff444f;
+    }
+    &__yellow-dark {
+        background: #b3760d;
+    }
+    &__yellow-light {
+        background: #ffa912;
+    }
+    &__violet-dark {
+        background: #4a3871;
+    }
+    &__brown-dark {
+        background: #664407;
```

Comment:
Add all these colour into the constant.scss file and check whether these colour already existed in the file itself